inst/schema/FormOptions.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "selectControl": {
+      "properties": {
+        "name": { "type": "string" },
+        "label": { "type": "string" },
+        "type": { "enum": [ "select", "multiselect" ] },
+        "required": { "type": "boolean" },
+        "default": { "type": "string" },
+        "helpText": { "type": "string" },
+        "options": { "type":  "array", "items": { "type": "string" }}
```

Comment:
Will we in fact need: 
* separate id and human readable name for options?
* nested options?

Reply:
Separate ID and human readable names - quite possibly this will depend on how the model is running but perhaps for safety we should do this i.e. does the model use quarter as ID or as the human readable text
Nested options - This could have some use in the region selector. In mrc-570 I've just returned a flat list of regions for the time being we could move this to a hierarchy and use a similar widget to region filters